resources/css/global.css

```diff
@@ -67,7 +67,8 @@ h2 {
 
 /*Main Nav-Bar*/
 .top-nav {
-  background-color: #FFC82B;
+  /*background-color: #FFC82B;*/
```

Comment:
I think that's a code smell, let's remove this comment

Reply:
It's not showing up as a code smell, and I wanted to keep the specific color in case we needed to re-implement it. It seems harmless to keep for now.

Reply:
ok sounds fair, we really wouldn't want to loose that specific shade of yellow

let's merge this in.